File: Frontend-Social-Party/src/app/modelos/EventoEntradaDTO.ts
import {EmpresaEntradaDTO} from "./EmpresaEntradaDTO";

export interface EventoEntradaDTO {
    id: number;
    titulo: string;
    fecha: string;
    horaApertura: string;
    horaFinalizacion: string;
    precio: number;
    empresa: EmpresaEntradaDTO;
}

File: Backend-Social-Party/src/main/java/org/example/backendsocialparty/DTOs/EventoEntradaDTO.java
package org.example.backendsocialparty.DTOs;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EventoEntradaDTO {
    private Integer id;
    private String titulo;
    private String fecha;
    private String horaApertura;
    private String horaFinalizacion;
    private Double precio;
    private EmpresaEntradaDTO empresa;
}
